src/components/common/animations/SpinnerCheckMark/SpinnerCheckMark.js

```diff
@@ -25,27 +34,29 @@ class SpinnerCheckMark extends React.Component {
   }
 
   componentDidUpdate(prevProps) {
-    if (prevProps.success === false && this.props.success === true && !isMobileReactNative) {
+    if (prevProps.success === false && this.props.success === true) {
       //speed up when finished
-      this.anim.setSpeed(1.5)
+      this.setState({
+        speed: 1.5,
+      })
     }
   }
 
   render() {
     const { height = 196, width = 196 } = this.props
-    if (!isMobileReactNative) {
-      animationData.layers[1].sc = '#ffffff00'
-    }
     return (
-      <Lottie
-        ref={this.setAnim}
-        source={animationData}
-        style={{
-          marginTop: -height / (isMobileReactNative ? 5 : 3),
-          width,
-          height,
-        }}
-      />
+      <View>
+        <Lottie
+          ref={this.setAnim}
+          source={animationData}
+          speed={this.state.speed}
```

Comment:
why do we also need this?

Reply:
@AlexeyKosinski waiting

Reply:
if u need to change speed of animation on ios/android we need this